### PR TITLE
feat: Update deps to 2.0.0 (#308)

### LINKED ISSUE #308: Update dependencies
cw, cw-utils, cw2, cw-storage-plus, cw-multi-test: 2.0

### PR DESCRIPTION
**Pull request by jawoznia**
close #308

### CODE REVIEW

File: sylvia/tests/replies.rs [hunk 1]
```diff
@@ -77,9 +79,13 @@ mod reply_contract {
 fn entry_point_generation() {
     let msg = Reply {
         id: 0,
+        payload: Default::default(),
+        gas_used: 0,
+        #[allow(deprecated)]
```
Comment (kulikthebird) [inline sylvia/tests/replies.rs:84]: Is there a way to update this test without adding `allow(deprecated)`?

File: README.md [hunk 2]
```diff
@@ -777,13 +777,13 @@ mod tests {
```
Comment (kulikthebird) [inline README.md:777]: Comment to other part of README:

Update the example project's deps (for e.g. `cosmwasm-std = "1.2.5"`) to the proper version. Also let's make sure that this corresponds with the `sylvia-template`

**Review by kulikthebird — CHANGES_REQUESTED** [review]
LGTM, only minor comments

File: sylvia/tests/replies.rs (hunk 1 above)
Comment (jawoznia) [reply sylvia/tests/replies.rs:84]: I think there is not. There is an issue about it https://github.com/rust-lang/rust/issues/102777, but no action so far.

**Pull request merged**